Sources/HttpServer/Requests/HttpRequest.swift

```diff
@@ -7,7 +7,7 @@ public class HttpRequest {
     var crlf: String
     var space: String
 
-    public init(method: HttpMethod?, url: String, version: String, headers: [String: String], body: [String: String]) {
+    public init(method: HttpMethod? = nil, url: String, version: String, headers: [String: String], body: [String: String]) {
```

Comment:
Why are you setting this to nil as a default? Doesn't every request need a method?

Reply:
probably easier to explain this in person. too much typing.